src/ScottPlot/Palette.cs

```diff
@@ -14,6 +14,7 @@ public static class Palette
         public static ScottPlot.Drawing.Palette Aurora => new(new ScottPlot.Drawing.Colorsets.Aurora());
         public static ScottPlot.Drawing.Palette Category10 => new(new ScottPlot.Drawing.Colorsets.Category10());
         public static ScottPlot.Drawing.Palette Category20 => new(new ScottPlot.Drawing.Colorsets.Category20());
+        public static ScottPlot.Drawing.Palette ColorBlindness => new(new ScottPlot.Drawing.Colorsets.ColorblindFriendly());
```

Comment:
I'm wondering if `ColorBlind` or `ColorBlindFriendly` is a better name than `ColorBlindness`? I'm not sure how to articulate why. Either name is fine with me, so go with whichever you prefer.

Reply:
Oops, good catch! I'll make this change and merge. Thanks!